TEST_WORKFLOW.md

```diff
@@ -0,0 +1,136 @@
+
+## 4	Repositories
+
+4.1	The following code and documentation repositories are used:
+- Master Test Repository - short name: X-Road-tests; hosted by: GitHub; managed by: RIA; purpose: release of X-Road automated tests; access: Head Architect has write access; read access: ALL.
+	- https://github.com/ria-ee/X-Road-tests
+
+4.2	Partners can establish their own, additional repositories, for backup, software distribution or other purposes.
+
+## 5	Branching pattern
+
+5.1	Branching pattern follows the [Gitflow model](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow). Two perpetual branches – `master` , `develop` and own `testfeature` branches.
+
+- `master` branch is used to release X-Road tests
+- `develop` branch is used to test development
+- `feature` branches are used to work on test features
+
+Main repository:
+- `master` branch https://github.com/ria-ee/X-Road-tests/tree/master
+	- releases of X-Road-tests that correspond to a specific release of X-Road source code. Several versions of X-Road-tests may point to the same X-Road source code version if necessary.
+	-  `master` branch releases are tagged with e.g "X-Road-tests for X-Road 6.16.0"
+- `develop` branch https://github.com/ria-ee/X-Road-tests/tree/develop
+	- Integration branch between Finland and Estonia (and other possible partners)
+	- Latest integrated test development source code 
+
+README.md file in the root of X-Road-tests must indicate the X-Road source code version for both MASTER and DEVELOP branches so that there is no confusion for which version these tests were designated for. Indication is done by pointing to the specific tag (MASTER) or commit (DEVELOP) of the X-Road source code.
+	
+	
+
+Fork repository:
+- `master` branch https://github.com/vrk-kpa/X-Road-tests and https://github.com/asaquality/X-Road-tests
+	- Tagged releases of X-road-tests
+- `develop` branch https://github.com/vrk-kpa/X-Road-tests and https://github.com/asaquality/X-Road-tests 
+	- Country branch for test development
+	- Latest country test development source code 
+	- Make pull request to develop for ria-ee/X-Road-tests
+
+## 6 Tagging
+
+6.1 The versions merged to X-Road-tests/master branch are tagged by the Head Architect.
+
+git tag -a 6.16.0 -m "X-Road-tests for X-road version 6.16.0"
+
+The versions merged to X-Road-tests/develop branch are not tagged.
+
+## 7 Changelog management and Pull requests
+
+Changelog description is visible in pull requests. Pull requests from Partners made against the `ria-ee/X-Road-tests/master` or `ria-ee/X-Road-tests/develop` branch MUST follow these conventions:
+
+Pull request name format is 'country-month-year-description':
+Country=origin of the pull request
+Month=Month when the pull request was created
+Year=Year when the pull request was created
+Description=(short) Description of the pull request
+
+e.g Finnish-08-2017-Description-of-pull-request, Estonian-07-2017-Description-of-pull-request
```

Comment:
For better reading and to show that this is example text, use Italic text:
 e.g _Finnish-08-2017-Description-of-pull-request_, _Estonian-07-2017-Description-of-pull-request_

Reply:
done